docs/contributing/development/continuous_integration.rst

```diff
@@ -353,3 +353,24 @@ Finally, the report is uploaded to the
 following the ``<pr>/<commit>`` folder structure. If the pipeline fails, also a report is 
 generated, but not necessarily gives useful debug information (depends on which step the
 pipeline has failed).
+
+
+TARDIS Carsus Compatibility Check
+=============================================
+The TARDIS Carsus Compatibility Check or the "Bridge" compares reference data 
+generated with different versions of Carsus. It consists of two jobs- a "carsus-build" job to 
+generate an atomic file with the latest version of Carsus and a "tardis-build" job 
+to generate a new reference data with it. These two reference data files are compared using the 
+`this notebook <https://github.com/tardis-sn/tardis-refdata/blob/master/notebooks/ref_data_compare_from_paths.ipynb>`_.
+The workflow has a ``workflow_dispatch`` event so that it can be triggered manually, but is also 
+triggered every week due to the "save-atomic-files" workflow. 
+The bridge fails currently there is a discrepancy between 
```

Comment:
I don't think it's worth including this information since it requires us to keep it up to date.

Reply:
Should we then have a general statement about the purpose of the workflow then? Or should I remove the entire added documentation for the bridge and the save-atomic-files workflow?

Reply:
No, the part you should remove is that "the bridge fails currently..." the documentation should just include the intended behaviour. Issues should list problems with that intended behaviour.

Reply:
I see!